wireguard-go-rust-wrapper/wireguard-go/main.go

```diff
@@ -97,6 +98,13 @@ func tracef(format string, args ...interface{}) {
 	trace_logger.Printf(format, args...)
 }
 
+type DevLoggerWriter struct {}
+
+func (t DevLoggerWriter) Write(p []byte) (n int, err error) {
+	debugf(string(p[:]))
```

Comment:
Does this only have a debug logs? Or can we differentiate?

Reply:
I'm afraid that it provides only redirection.  It seems that wintun has only INFO and ERROR logs and I felt like making them DEBUG makes some sense, replacing both of them with DEBUG sounds better to me than making INFO -> ERROR or ERROR -> INFO (they are not our logs and we add them now mostly for debugging purposes anyway).

Reply:
Yeah, maybe we can leave the `debug` only, but maybe than mark them as `wintun-log` or sth similar? Do distinguish them from the rest of the logs.

Reply:
@matislovas added some tag